scripts/patrol/patrol.py

```diff
@@ -618,6 +618,7 @@ def generate_patrol_events(self, patrol_dict):
                 biome=patrol.get("biome"),
                 season=patrol.get("season"),
                 tags=patrol.get("tags"),
+                weight=patrol.get("weight"),
```

Comment:
```suggestion
                weight=patrol.get("weight", 20),
```

The default one is 20 so it should that and not 'None'